### DISCUSSION TITLE
Texture format for "A/+ button"

### DISCUSSION THREAD

**Discussion by User670** [chrislo27/PolyrhythmMania#28] [Ideas]
Seems like currently "A", "bo", "ta", "n" are separate textures. Does this mean if someone wants to change the text here, eg. make it read "PUSH A" like in Rhythm Heaven Silver, it would be difficult or impossible to do? Or is it intended that this is always left in Japanese?

ps. this is not an "idea", but "idea" is the only category that is available to me. If you disallow non-ideas to be posted here let me know?

**Comment by chrislo27** [comment]
It is intended that the button signs remain in Japanese. *Rhythm Heaven Silver* is an unofficial fan translation of Rhythm Tengoku. I do not rely on its work nor have I referenced it in developing *Polyrhythm Mania*.

**Comment by chrislo27** [comment]
I just thought of something: you can set the button sign textures to be fully transparent, and use the various Middleground/Foreground Image textures to emulate the signs however you need them to look.
